shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -82,6 +125,7 @@ - (instancetype)initWithEngine:(FlutterEngine*)engine
     _flutterView.reset([[FlutterView alloc] initWithDelegate:_engine opaque:self.isViewOpaque]);
     _weakFactory = std::make_unique<fml::WeakPtrFactory<FlutterViewController>>(self);
     _ongoingTouches = [[NSMutableSet alloc] init];
+    _ioWorkerPause.reset(new TaskRunnerPause([_engine.get() IOTaskRunner]));
```

Comment:
I think we can get here before the engine finishes creating the shell, which would either give us nil for the task runner or throw an exception for trying to access it before the shell has set up.

Reply:
Good catch, addressed.